recipes/sanja/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "sanja" %}
+{% set version = "1.0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b6b87c2a2780abde59bdd6e13bcfe91f11d4217a192f1f401f83312a6b145420
+
+build:
+  #noarch: python
+  number: 0
+  skip: True  # [py<38]
```

Comment:
This can be noarch python with `python >=3.8` below

Reply:
At first I had:

    build:
        noarch: python
        number: 0
        skip: True  # [py<38]
        script: "{{ PYTHON }} -m pip install . -vv"

    requirements:
        host:
            - python
            - pip
    run:
        - python >=3.8
        - sanic
        - jinja2

But I got 
> noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python

Reply:
The last one